atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -38,7 +38,11 @@
         while (true) {
             checkOpen();
             try {
-                return runTaskThrowOnConflict(task);
+                T result = runTaskThrowOnConflict(task);
+                if (failureCount > 0) {
+                    log.info("Successfully completed transaction after {} retries.", failureCount);
```

Comment:
This is somewhat covered by the traceId's being logged out.  That said, a particular span could be running multiple transactions, but at least the success should follow a single failure for a particular traceId in single threaded AtlasDB client work flows.